sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -22,11 +22,20 @@
 #endif
 
 #ifdef __SYCL_DEVICE_ONLY__
+
+#if SYCL_EXT_ONEAPI_MATRIX == 1
```

Comment:
Could you please clarify where this macro is defined? I've got an impression that the macro should be defined to some specific value in our headers.

Reply:
The macro is defined here: https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/feature_test.hpp.in